src/account-link.ts

```diff
@@ -1,6 +1,10 @@
 import prefixForNetwork from './prefix-for-network'
 
-export = function getAccountLink(address: string, network: string): string {
-  const prefix = prefixForNetwork(network)
-  return prefix !== null ? `https://${prefix}etherscan.io/address/${address}` : '';
+export = function getAccountLink(address: string, network: string, customNetworkUrl?: string): string {
+  if(customNetworkUrl?.length){
+    return `${customNetworkUrl}/address/${address}`;
```

Comment:
It would be better to use the `URL` API to parse the URL, so that you can ensure this path still makes sense. This would be problem if the custom network URL passed in included query string parameters or a hash, or if it ended in a slash already.

With the URL API you can deconstruct this into its pieces, check the pathname for a trailing slash and add one only if it's missing, add the rest of the path, then reconstruct the URL again preserving all query string parameters. That would allow the caller to pass in important information like the API key, for any block explorer that accepts it via query string anyway.